src/Microsoft.Framework.PackageManager/Scripts/ScriptExecutor.cs

```diff
@@ -97,9 +97,9 @@ public bool Execute(Runtime.Project project, string scriptName, Func<string, str
                         scriptArguments[0] = $"{ quote }{ prefix }{ scriptCandidate }{ quote }";
                     }
 
-                    // Always use /bin/bash -c in order to support redirection and so on; similar to Windows case.
+                    // Always use /usr/bin/env bash -c in order to support redirection and so on; similar to Windows case.
                     // Unlike Windows, must escape quotation marks within the newly-quoted string.
-                    scriptArguments = new[] { "/bin/bash", "-c", "\"" }
+                    scriptArguments = new[] { "/usr/bin/env bash", "-c", "\"" }
```

Comment:
First element in array needs to be split in two. `/usr/bin/env` is the executable while `bash` is the first argument.